spec/request/posts_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/users/:user_id/posts'
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/users/:user_id/posts/:id'
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```

Comment:
- [ ] Great job here 💯 However, according to the requirements, your file should include a test that ensures correct view template is rendered and a test that ensures the response body includes the correct placeholder text 👍

> For Example  💡
```ruby
    it 'Check if a correct template is rendered' do
      expect(response).to render_template(:show)
    end
```
```ruby
    it 'Check if the response body includes correct placeholder text' do
      expect(response.body).to include('A specific post for a specific user')
    end
```

![image](https://user-images.githubusercontent.com/45231276/202047666-d7e8bf43-56d2-40af-8e82-9c87f7fb3959.png)